actions-support/ci-image/Dockerfile

```diff
@@ -0,0 +1,98 @@
+# syntax=docker/dockerfile:1-labs
+# Download and unpack the toolchain on the build host architecture
+FROM --platform=$BUILDPLATFORM ubuntu:latest as toolchain-unpacker
+ARG BUILDPLATFORM
+ARG DBT_VERSION
+
+# Install tools required to unpack the toolchain archive
+RUN apt-get update && apt-get install -y curl parallel
+
+# Use the local toolchain files
+COPY ./dist/dbt-toolchain-$DBT_VERSION-linux-arm64.tar.gz /dbt/toolchain/
+COPY ./dist/dbt-toolchain-$DBT_VERSION-linux-x86_64.tar.gz /dbt/toolchain/
+
+# Unpack the toolchain in the unpack container
+RUN <<LOAD_TOOLCHAIN bash
+  set -ex
+  mkdir -p /dbt/toolchain
+  cd /dbt/toolchain
+  targets="x86_64 arm64"
```

Comment:
For DBT-Toolchain we /probably/ don't need both architectures in the automation, but probably doesn't hurt.